storage/redis_controller.go

```diff
@@ -3,44 +3,55 @@ package storage
 import (
 	"context"
 	"sync/atomic"
+	"time"
 
-	"github.com/TykTechnologies/tyk/config"
 	redis "github.com/go-redis/redis/v8"
 
-	"time"
+	"github.com/TykTechnologies/tyk/config"
 )
 
 type RedisController struct {
-	singlePool          atomic.Value
-	singleCachePool     atomic.Value
-	singleAnalyticsPool atomic.Value
```

Comment:
We should not use atomic.Value for redis.UniversialClient interface + *v8.Redis pointers. Removing.